src/main/resources/database/changelog/changelog-tables.xml

```diff
@@ -0,0 +1,35 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="create users table" author="Alex">
+        <createTable tableName="app_user">
+            <column name="user_id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="customer_id" type="varchar(32)"/>
+        </createTable>
+        <createTable tableName="invoice">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="invoice_id" type="varchar(32)">
```

Comment:
what is the difference between `id` and `invoice_id`?

Reply:
While we use Stripe Payment system, there is no difference. (`id `- auto increments identifier, `invoice_id `- is id from Stripe that is used for retrieve invoice in Stripe and to pay it).  `id` is here for future when will be added one more payment system, but now we can remove it